### PR TITLE
[fix](inverted index) fix error handling in fast_execute

### PR DESCRIPTION
**Pull request by zzzxl1993**
## Proposed changes

1. fix error handling in fast_execute
2. Fixed the issue where fast_execute was still being executed when set enable_inverted_index_query = false.

### CODE REVIEW

**Comment by zzzxl1993** [comment]
run buildall

**Review by csun5285 — APPROVED** [review]
LGTM

File: be/src/vec/exprs/vexpr.cpp [hunk 1]
```diff
@@ -651,11 +656,19 @@ std::string VExpr::gen_predicate_result_sign(Block& block, const ColumnNumbers&
...
             for (size_t i = 1; i < arguments.size(); i++) {
                 const auto& entry = block.get_by_position(arguments[i]);
+                if (!is_column_const(*entry.column)) {
+                    return "";
+                }
                 values.insert(entry.type->to_string(*entry.column, 0));
             }
             pred_result_sign += boost::join(values, ",");
+        } else if (function_name == "collection_in" || function_name == "collection_not_in") {
+            return "";
```
Comment (airborne12) [inline be/src/vec/exprs/vexpr.cpp:666]: When will function_name be collection_in?

**Review by airborne12 — DISMISSED** [review]
LGTM

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Comment by zzzxl1993** [comment]
run buildall

**Review by qidaye — APPROVED** [review]

**Pull request merged**